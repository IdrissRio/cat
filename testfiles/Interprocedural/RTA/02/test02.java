public class test02 {
  public void m(){};
  static void main(String... args) { new test02().m(); }
}
