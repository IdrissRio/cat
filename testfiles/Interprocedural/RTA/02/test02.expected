test02::Static:{
}
test02::Instance:{
}
test02::m:{
}
test02::main:{
	test02::m
}
